public/v4/apps/invidious.yml

```diff
@@ -0,0 +1,58 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: postgres:$$db_version
+        restart: always
+        volumes:
+            - $$cap_appname-data:/var/lib/postgresql/data
+        environment:
+            POSTGRES_DB: invidious
+            POSTGRES_PASSWORD: kemal
+            POSTGRES_USER: kemal
+        healthcheck:
+            test: ['CMD', 'pg_isready', '-U', 'postgres']
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: omarroth/invidious:$$cap_version
+        restart: always
+        environment:
+            # Adapted from ./config/config.yml
+            INVIDIOUS_CONFIG: |
+                channel_threads: 1
+                check_tables: true
+                feed_threads: 1
+                db:
+                  user: kemal
+                  password: kemal
+                  host: srv-captain--$$cap_appname-db
+                  port: 5432
+                  dbname: invidious
+                full_refresh: false
+                https_only: false
+                domain:
+        caproverExtra:
+            containerHttpPort: '3000'
+
+caproverOneClickApp:
+    variables:
+        - id: '$$cap_version'
+          label: Invidious Version
+          defaultValue: 'latest' # there is no other tag provided
```

Comment:
Can you try using the sha tag?

Reply:
I can do that, it works. (I am not entirely sure if it is better that way, since I imagine that invidious updates fairly often, because of changes youtube does.)

Reply:
Your comment in the variable label is pretty self explanatory. Thanks!